addon/hint/show-hint.js

```diff
@@ -229,14 +229,17 @@
       elt.hintId = i;
     }
 
+    var container = completion.options.container || ownerDocument.body;
+    var positionOffset = (container.offsetParent || ownerDocument.body).getBoundingClientRect();
     var pos = cm.cursorCoords(completion.options.alignWithWord ? data.from : null);
-    var left = pos.left, top = pos.bottom, below = true;
+    // We offset the cursor position because left and top are relative to the offsetParent's top left corner.
+    var left = pos.left - positionOffset.x, top = pos.bottom - positionOffset.y, below = true;
```

Comment:
`.x` and `.y` don't exist on IE and Edge, so you'll want to use `.left` and `.top` instead.